config/exporter.go

```diff
@@ -25,8 +25,8 @@ type Exporter interface {
 type Exporters map[ComponentID]Exporter
 
 // ExporterSettings defines common settings for an exporter configuration.
-// Specific exporters can embed this struct and extend it with more fields if needed.
-// When embedded in the exporter config it must be with `mapstructure:",squash"` tag.
+// Specific exporters can embed this struct, and extend it with more fields if needed.
+// When embedded in the exporter config, it must be with `mapstructure:",squash"` tag.
```

Comment:
```suggestion
// Specific exporters can embed this struct and extend it with more fields if needed.
// When embedded in the exporter config it must be with `mapstructure:",squash"` tag.
```

I don't think the commas are necessary here.

Reply:
I agree that the first sentence does not need the comma (parallel verbs), but I believe it wouldn't hurt readability to have the second comma (Dependent clause, Independent clause)!
```suggestion
// Specific exporters can embed this struct and extend it with more fields if needed.
// When embedded in the exporter config, it must be with `mapstructure:",squash"` tag.
```